src/components/accounts/two_factor/EnableTwoFactor.js

```diff
@@ -166,7 +176,7 @@ export function EnableTwoFactor(props) {
                     >
                         <Translate id={`button.continue`} />
                     </FormButton>
-                    <FormButton className='link' linkTo='/profile'><Translate id='button.skip' /></FormButton>
+                    <FormButton className='link' onClick={() => skipUrl()}><Translate id='button.skip' /></FormButton>
```

Comment:
In this case you just need use `dispatch(redirectToApp('/profile'))` it will redirect to `/login`